packages/countWords/README.md

```diff
@@ -0,0 +1,24 @@
+# `@plexis/count-words`
+
+Returns the number of words contained in the input text.
+
+**Installation**
+`npm i @plexis/count-words`
+
+**Usage**
+
+```javascript
+import countWords from '@plexis/count-words';
+
+countWords(); // returns 0
+countWords(''); // returns  0
+countWords('     '); // returns 2
```

Comment:
Should be

`countWords('     '); // returns 0`

Reply:
done